ansible/roles/cartodb.postgresql/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+postgresql_version: 9.5
+postgresql_listen_addresses: localhost
+postgresql_port: 5432
+postgresql_data_directory: /var/lib/postgresql/{{ postgresql_version }}/main
+postgresql_max_connections: 100
+postgresql_shared_buffers: "128MB"
+postgresql_work_mem: "4MB"
+postgresql_log_autovacuum_min_duration: -1
+postgresql_log_min_duration_statement: -1
```

Comment:
Might be worth setting this to something like `500`. Doubt you're at the point where tuning queries is a main concern, but could be useful if you happen to run into something and then have the slow queries already in the log.